scripts/eosjs-port.js

```diff
@@ -104,19 +104,27 @@ class Eos {
 
         const data = {}
 
-        if (action.fields.length > 0 && arguments.length == 2 && typeof arguments[0] === 'object') {
-          for (let i = 0; i < action.fields.length; i++) {
-            const { name } = action.fields[i]
-            data[name] = arguments[0][name]
-          }
-        } else {
-          if ((action.fields.length + 1) != arguments.length) {
-            throw new Exception(`Not enough arguments to call ${action.name} action in ${accountName} contract`)
-          }
-          for (let i = 0; i < action.fields.length; i++) {
-            const { name } = action.fields[i]
-            data[name] = arguments[i]
-          }
+        // if (action.fields.length > 0 && arguments.length == 2 && typeof arguments[0] === 'object') {
```

Comment:
What about this commented out code? Replaced by new simpler code?

Reply:
Yes, as I had to pass an array now for calling the functions I updated the way it was handling the parameters.